server/repository/assessment/index.ts

```diff
@@ -16,6 +18,8 @@ export const AssessmentRepository = {
   createAssessmentSchema,
   read,
   readSections,
+  readTablesMetadata,
```

Comment:
this would be more readTables (see readSections for metadata conventions. use the same)